OreshkoJA/lab1

```diff
@@ -0,0 +1,39 @@
+#include <iostream>
+using namespace std;
+int main()
+{
+	setlocale(0, "");
+	float first, second;
+	cout << "Введите первое число:";
+	cin >> first;
+	char sign;
+	cout << "Введите знак:";
+	cin >> sign;
+	cout << "Введите второе число:";
+	cin >> second;
+	float result;
+	if (second == 0) {
+		if (sign == '/') {
+			cout << "Делить на ноль нельзя" << endl;
+			system("pause");
+		}
+	}
+	else {
+		switch (sign) {
+		case '+': result = first + second;
+			break;
+		case '-': result = first - second;
+			break;
+		case '*': result = first * second;
+			break;
+		case '/': result = first / second;
+			break;
+		default: cout << "Неправильный ввод знака" << endl;
+			system("pause");
+			return 0;
```

Comment:
Do not use return statement here, as it would immediately shut down the application. In current work terms it's OK, but normally you don't want your calculator to shut down after every mistake you made. Try to solve it in another way.